starter-code/server.js

```diff
@@ -13,18 +16,17 @@ const app = express();
 // Windows and Linux users; You should have retained the user/pw from the pre-work for this course.
 // Your url may require that it's composed of additional information including user and password
 // const conString = 'postgres://USER:PASSWORD@HOST:PORT/DBNAME';
-const conString = 'postgres://localhost:5432';
+const conString = 'postgres://maks@localhost:5432/lab08'; //check on this
 
```

Comment:
So instead of 'lab08' at the end, you want to connect to the name of your database (blog?)